x/tokenfactory/module.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/spf13/cobra"
 	abci "github.com/tendermint/tendermint/abci/types"
 
+	appparams "github.com/osmosis-labs/osmosis/v10/app/params"
+
```

Comment:
Sorry I didn't notice this in my first pass - we shouldn't have modules depend on the app params. The simulator genesis state should just hardcode a denom / amount for now, rather than replace what the app params base denom is

Reply:
Default genesis defaults to uosmo, so I just wanted to replace uosmo (BaseCoinUnit) with stake (DefaultBondDenom) without hardcoding either. Should I remove this and instead just hardcode "uosmo" in place of appparams.BaseCoinUnit?

Reply:
Ref this right here https://github.com/osmosis-labs/osmosis/blob/88125a37830035ef7c9222ce36f8bfc6aed2aedc/x/tokenfactory/module.go#L195-L199

Reply:
Just hardcode the new DenomCreationFee

Reply:
Theres not really any point in caring about the default setting. The space that makes sense is:
- Random coin / coins
- hardcoded constant

Generically adapting the default doesn't really matter.